opendevin/core/config.py

```diff
@@ -51,13 +51,14 @@
     ConfigType.AGENT: 'CodeActAgent',
     ConfigType.E2B_API_KEY: '',
     ConfigType.SANDBOX_TYPE: 'ssh',  # Can be 'ssh', 'exec', or 'e2b'
-    ConfigType.USE_HOST_NETWORK: 'false',
+    ConfigType.USE_HOST_NETWORK: False,
     ConfigType.SSH_HOSTNAME: 'localhost',
-    ConfigType.DISABLE_COLOR: 'false',
+    ConfigType.DISABLE_COLOR: False,
```

Comment:
@SmartManoj Definitely these are good catches. Have you seen https://github.com/OpenDevin/OpenDevin/pull/1552 ? It's knocking them out, because they get type hints, and mypy isn't letting any of them pass anymore. 😅